src/Controller/AuthenticationController.php

```diff
@@ -55,7 +55,7 @@ public function redirectAtLogin(Request $request, array $requestAttributes): Res
             $body = Stream::createFromString('');
             $response = new Response($body, '1.1', 301, '');
             /** @var Response $redirect */
-            $redirect = $response->withHeader('Location', 'http://local.quiz.com/candidate/homepage?page=1');
+            $redirect = $response->withHeader('Location', 'http://local.quiz.com/candidate/homepage');
             return $redirect;
```

Comment:
just return, no need to store it in another variable

Reply:
@Raresmldvn  All functions that redirect a response do it this way. I will make a separate issue for this one in which I want to make a function/class in mvc-framework to help me redirect responses

Reply:
he means that you can return $response->withHeader('Location', 'http://local.quiz.com/candidate/homepage') from the method, without storing the result in the $redirect variable first :)